cloud/ecs-architecture.md

```diff
@@ -1,3 +1,10 @@
+---
+title: ECS integration architecture
+description: Mapping of Docker compose entities to Amazon constructs
+keywords: Docker, Amazon, Integration, ECS, Compose, architecture, mapping
+toc_min: 1
+toc_max: 2
+---
```

Comment:
We need to add this front-matter to the upstream repository now

Probably the `toc_min` and `toc_max` can be skipped

Reply:
Added for all files, now merged in compose-cli main branch